hvpy/io.py

```diff
@@ -0,0 +1,29 @@
+from enum import Enum, auto
+
+from pydantic import BaseModel
+
+BASE_URL = "https://api.helioviewer.org/v2/"
+
+
+class HvpyParameters(BaseModel):
+    def dict(self):
+        d = super().dict()
+        # Pydantic doesn't allow using lower case 'json' as a field,
+        # so override it here.
+        if "Json" in d:
+            d["json"] = d["Json"]
+            del d["Json"]
+        return d
+
+    def get_output_type(self):
+        # Default output type is raw.
+        return OutputType.Raw
+
+    def url(self):
```

Comment:
```suggestion
    @property
    def url(self):
```
I think will allow us to avoid having to call the url.

Reply:
Throws an error: 
```
    def execute_api_call(input_parameters: HvpyParameters):
    
>       url = input_parameters.url()
E       TypeError: 'str' object is not callable
```

Reply:
Yes. You will need to change that line to not call a string.